src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -1527,6 +1527,16 @@ void Session::cancel_resumption_delay()
     if (unbind_process_complete())
         initiate_rebind(); // Throws
 
+    try {
+        process_pending_flx_bootstrap(); // throws
+    }
+    catch (const IntegrationException& error) {
+        on_integration_failure(error);
```

Comment:
what happens if we have an integration failure here? I guess the client will just continue to resume the session but without applying the bootstrap?

Reply:
Yes - it's the same if there is an integration failure during `activate()`